app/serializers/compliance_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# JSON serializer parent class
+module ComplianceSerializer
```

Comment:
Nit: It's not a class :laughing: But you could make it one if you'd like! `ApplicationSerializer` would be the rails pattern.